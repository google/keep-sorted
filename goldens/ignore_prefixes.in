Already sorted:
  // keep-sorted-test start ignore_prefixes=A,B
  Bar
  Baz
  Aoo
  zebra
  // keep-sorted-test end

Needs sorting:
  // keep-sorted-test start ignore_prefixes=A,B
  zebra
  Aoo
  Bar
  Baz
  // keep-sorted-test end

Indented already sorted:
  // keep-sorted-test start ignore_prefixes=A,B
  Bar
  Baz
  Aoo
  zebra
  // keep-sorted-test end

Indented needs sorting:
  // keep-sorted-test start ignore_prefixes=A,B
  zebra
  Aoo
  Bar
  Baz
  // keep-sorted-test end

More practical already sorted:
  // keep-sorted-test start ignore_prefixes=fs.setBoolFlag,fs.setIntFlag
	fs.setBoolFlag("nvme_paws_with_cute_toebeans", true)
	fs.setIntFlag("nvme_pretty_whiskered_kitten", 6)
	fs.setBoolFlag("nvme_whiskered_adorable_dog", true)
  // keep-sorted-test end

More practical needs sorting:
  // keep-sorted-test start ignore_prefixes=fs.setBoolFlag,fs.setIntFlag
	fs.setBoolFlag("nvme_paws_with_cute_toebeans", true)
	fs.setBoolFlag("nvme_whiskered_adorable_dog", true)
	fs.setIntFlag("nvme_pretty_whiskered_kitten", 6)
  // keep-sorted-test end

Combine with numerical:
  # keep-sorted-test start numeric=yes ignore_prefixes=R2D2,C3PO,R4
  R2D2_BOLTS_5_MM,
  R2D2_BOLTS_10_MM,
  R2D2_PROJECTOR,
  C3PO_ARM_L,
  C3PO_ARM_R,
  C3PO_HEAD,
  R4_MOTIVATOR,
  # keep-sorted-test end

Prefixes with spaces:
  # keep-sorted-test start ignore_prefixes=['* ', '* [']
  * foo
  * bar
  * [baz](path/to/baz)
  # keep-sorted-test end
