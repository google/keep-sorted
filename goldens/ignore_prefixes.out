Already sorted:
  // keep-sorted-test start ignore_prefixes=A,B
  Bar
  Baz
  Aoo
  zebra
  // keep-sorted-test end

Needs sorting:
  // keep-sorted-test start ignore_prefixes=A,B
  Bar
  Baz
  Aoo
  zebra
  // keep-sorted-test end

Indented already sorted:
  // keep-sorted-test start ignore_prefixes=A,B
  Bar
  Baz
  Aoo
  zebra
  // keep-sorted-test end

Indented needs sorting:
  // keep-sorted-test start ignore_prefixes=A,B
  Bar
  Baz
  Aoo
  zebra
  // keep-sorted-test end

More practical already sorted:
  // keep-sorted-test start ignore_prefixes=fs.setBoolFlag,fs.setIntFlag
	fs.setBoolFlag("nvme_paws_with_cute_toebeans", true)
	fs.setIntFlag("nvme_pretty_whiskered_kitten", 6)
	fs.setBoolFlag("nvme_whiskered_adorable_dog", true)
  // keep-sorted-test end

More practical needs sorting:
  // keep-sorted-test start ignore_prefixes=fs.setBoolFlag,fs.setIntFlag
	fs.setBoolFlag("nvme_paws_with_cute_toebeans", true)
	fs.setIntFlag("nvme_pretty_whiskered_kitten", 6)
	fs.setBoolFlag("nvme_whiskered_adorable_dog", true)
  // keep-sorted-test end

Combine with numerical:
  # keep-sorted-test start numeric=yes ignore_prefixes=R2D2,C3PO,R4
  C3PO_ARM_L,
  C3PO_ARM_R,
  R2D2_BOLTS_5_MM,
  R2D2_BOLTS_10_MM,
  C3PO_HEAD,
  R4_MOTIVATOR,
  R2D2_PROJECTOR,
  # keep-sorted-test end

Prefixes with spaces:
  # keep-sorted-test start ignore_prefixes=['* ', '* [']
  * bar
  * [baz](path/to/baz)
  * foo
  # keep-sorted-test end
